src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -34,11 +34,11 @@ protected override async Task TestWorkerAsync(Document document, TextSpan textSp
             if (mode == TestMode.Position)
             {
                 int position = node?.SpanStart ?? textSpan.Start;
-                inferredType = typeInference.InferType(await document.GetSemanticModelForSpanAsync(new TextSpan(position, 0), CancellationToken.None), position, objectAsDefault: true, cancellationToken: CancellationToken.None);
+                inferredType = typeInference.InferType(await document.ReuseExistingSpeculativeModelAsync(position, CancellationToken.None), position, objectAsDefault: true, cancellationToken: CancellationToken.None);
```

Comment:
introduced overload for this common pattern of passing in a position.

Reply:
Good question.  The answer currently is: we need this.  Specifically, this is used by VB's statement-cleanup auto fixers.  i.e. when you move off a line and we go case-correct things and whatnot. 

The primary issue there is that we have a a few layers of disconnect between the vb cleanup feature and the eventual cleanup-providers.  This eventually bubbles down to a set of spans to cleanup (primarily because we're cleaning annotated nodes).  Deep in the bowels we get that set of spans, create a single span representing all of it, and then say "as long as this is all within a method, we can do the reuse".  If there were less indirections and whatnot here, i think we could change it up.  i.e. have the caller figure out hte nearest containing node that contains all teh spans and pass that through.  But it's a bit too much work for this PR IMO.